package.json

```diff
@@ -1464,6 +1464,21 @@
           "default": true,
           "markdownDescription": "Show links to matching help pages in hover"
         },
+        "r.helpPanel.clickCodeExamples": {
+          "type": "string",
+          "default": "Copy",
+          "markdownDescription": "What happens when clicking code examples on help pages.",
+          "enum": [
+            "Disabled",
+            "Copy",
+            "Run"
```

Comment:
It is nice to have the capability to either copying or running the example code. I wonder if it is possible to have both at the same time. For example, click to copy, and cmd+click to run, or something like this?

Reply:
Good idea. I wasn't sure how to handle the possible [modifier keys](https://github.com/reditorsupport/vscode-r/blob/23aa7ca821956e6dd86330e3b44789c800b28431/html/help/script.ts#L100-L103) best, especially w.r.t. mac (Should we just make them all configurable, or are there sensible "groupings"?).